app/views/comments/_comment.html.erb

```diff
@@ -7,17 +7,19 @@
       <h5 class="mt-0">
         <%= link_to comment.author.username, user_path(comment.author.username), class: "text-dark link-underline-dark link-underline link-underline-opacity-0 link-underline-opacity-100-hover" %>
       </h5>
+      console.log("this is working")
```

Comment:
Careful leaving the logs in once you know its working, definitely don't want to deploy like that!